packages/server/src/api/controllers/table/index.js

```diff
@@ -49,6 +49,7 @@ exports.fetch = async function (ctx) {
       ...entity,
       type: "external",
       sourceId: row.doc._id,
+      sql: isSQL(row.doc),
```

Comment:
Is the variable naming right here? I see above we query datasources and call it `externalTables`, should that be `datasources`? would make more sense here as `isSql` expects a datasource and not a table. Might help if we rename the `row` here to `datasource` as row can be confused with a table row which was my first thought

Reply:
Yeah, this is just a very old function, dates back to before data was ever called rows, I'll fix it!